test/jdk/javax/management/monitor/ThreadPoolAccTest.java

```diff
@@ -151,6 +163,9 @@ private static boolean testPrincipals(ObservedObject[] monitored, ObjectName[] m
         for (int i = 0; i < 6; i++) {
             String principal =  monitored[i].principal;
             String expected = principals[i / 3];
+
+            echo("testPrincipals: monitored: " + monitored[i] + " principal: " + principal + " expected: " + expected);
```

Comment:
With this in place, should printing of principal at line 176 be removed?

Reply:
Right, it is duplicated.  That new line is useful, I really wanted to see something before we check for null.
Then we have the existing printing before we check further, which I did not disturb.  I'm sure it's not a big deal either way.

Reply:
Up to you.